resources/environments.php

```diff
@@ -27,6 +28,7 @@
     ],
     'pre' => [
         'base_url' => 'https://dev.kom-inn.org',
+        'logfile' => '/var/log/kominn.log',
```

Comment:
`/var/log/kom-inn-xxx.log` to be consistent with other logs(`/var/log/kom-inn-emails.log`). Same for pro